### ISSUE TITLE
Fuel rebalancing

### ISSUE THREAD

**Issue by D-Alessian** [m2r1k5/nomi-ceu#1153]
### Nomi CEu Version

Nightly

### Launcher

Prism

### Configurations or Mods Changed

N/A

### Environment

N/A

### Mode

Both Modes

### Related Problem

Fuel should be a better powergen option but it simply isn't right now. Only HOG is decent, but still outclassed, and in HM fuel is good only because there aren't a lot of options. I consider Fusion to be an extension of fuel too so buffs made to fuel could apply there.

### Your Solution

There are many solutions possible:
1. Simply buff the power density. This is the easiest but also least interesting and probably least effective buff possible, but it probably should be used in conjuction with other options
2. Make turbine blade renewal automatable. Currently the best way is to have a big monitor with all your turbines and do it from there manually, but it's still a huge disadvantage. You could also buff the longevity by a huge factor, i don't feel like longevity is really an interesting gameplay element if it's a matter of hours. In GTNH blades last literal months. You could maybe make it so that blades get a lot more durability used if running at low powers for example, or if maintenance isn't done, or add a lubricant slot needed for them to take 1/20th the durability.
3. Make Turbines and CEs just output more power. Keep power density the same but reduce the absolutely insane spam needed for using them.
4. Give more tiers and variations of Turbines and CEs. Add a bigger ECE that can do the work of 30 of them, using 90% of the fuel they would use and 110% the power for example. Add Turbines needing multiple blades with some kind of interesting dynamic. Just CEs and Turbines aren't enough to make fuel viable.

### How Will Your Solution Work

Both changes to Nomilabs for the multiblocks, maybe some mixins if changing fundametally the Turbines/CEs and changes to Nomi-CEu probably in GS for changing power density.

### Alternatives You've Considered

Buff HOG by x16384k the power density please

**Comment by Folanlron** [comment]
Turbine replacement is a CEu problem not a pack problem, even NH literally just got the ability to replace them automatically like a little over a year ago. (The hatch/robot system had been in code sense gt5 was initially released, but was never finished just like the rest of GT5.)

**Comment by D-Alessian** [comment]
> Turbine replacement is a CEu problem not a pack problem
Yes and no. Nomi can implement changes to GregTech either with mixins or with custom blocks. Turbines being made in CEu doesn't mean they can't be changed in Nomi.

**Comment by D-Alessian** [comment]
> Referring to 2, making turbine blade renewal automatable wouldn't be that useful. On average, blades last 60h+, in a regular playthrough, you would only need to replace them once or twice at most.
 
On servers it would make a huge difference. 

> As for 4: a bigger ECE would be incredibly broken, probably even outclassing solar panels, a similar situation also applies to plasma turbines. The main problem with power is the dip between mid HV, where HV combustion engines starts to not scale well, and late EV, where you get LCEs. For example, we built 48 HV combustion engines in our last playthrough.

I don't know, would it really? You can make that bigger ECE much more expensive. I just know that to get a reasonable LuV power setup you need about 100-300 ECEs, that doesn't feel quite right.

> As for the late game power problem. It's mostly the fault of solar panels, which are too op, besides that, plasma turbines with helium plasma are an incredible source of power. So I don't think a mult-multiblock plasma turbines would be necessary.

You still need 80 turbines to max out a T3 reactor for example, I still think that's way too much and not an interesting gameplay loop. I'd rather build one huge expensive multi than 80 boring small multis. You could even make the big turbines alternative worse than spamming them individually but multi spam doesn't feel very Nomic

**Comment by Folanlron** [comment]
> Yes and no. Nomi can implement changes to GregTech either with mixins or with custom blocks. Turbines being made in CEu doesn't mean they can't be changed in Nomi.


I would rather see a proper solution from the CEu team, then some mixin/mod that could cause massive problems, when they do add it.
